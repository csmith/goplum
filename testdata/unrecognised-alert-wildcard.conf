alert debug.sysout "alert1" {}

check debug.random "c3" {
    alerts = "b*"
    interval = 3s
    good_threshold = 5
    failing_threshold = 6
}
