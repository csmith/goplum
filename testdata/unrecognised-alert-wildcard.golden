"error configuring check c3: no alerts match 'b*'"